Lesson20/selcan/oop.js

```diff
@@ -0,0 +1,152 @@
+// 1. Create a Person class with name, age, and gender properties, and a greet() method that logs a greeting message to the console. Then create a Student class that extends the Person class and adds a major property. Override the greet() method in the Student class to log a different greeting message to the console. Create 2 variables, one of type Person and one of type Student and log them to console.
+
+class Person {
+    constructor(name,age,gender){
+        this._name = name;
+        this._age = age;
+        this._gender = gender;
+    }
+    greet(){
+        return `Hi!My name is ${this._name}.I'm ${this._age} years old.I'm a ${this._gender}.`
+    }
+}
+class Student extends Person {
+    constructor(name,age,gender,major){
+        super(name,"yüzmeyi çok sever.",gender);
```

Comment:
The second value should be age variable (so a number type) but it's not a string "yüzmeyi çok sever." which does not make sense. Please change to a number
```suggestion
        super(name, age, gender);
```